monitoring/README.md

```diff
@@ -86,14 +86,20 @@ are present in `USER_DIR`:
 * `monitoring/user-values-prom-operator.yaml`
 * `monitoring/user-values-pushgateway.yaml`
 
+You can modify two types of values to customize your deployment:
+- Environment variables: These variables contain uppercase characters and underscores (for example, `LOG_COLOR_ENABLE=`), and can be specified in the `user.env` file or on the command line. Specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments.
+- Helm chart parameters: These parameters are organized into a hierarchical structure and contain mixed case characters (for example, `persistentVolume:storageClass`). These parameters are specified in `.yaml` configuration files.
+
 ### user.env
 
-The `monitoring/user.env` file contains environment variable flags that customize
-the components that are deployed or to alter some script behavior (such as to
+The `monitoring/user.env` file contains environment variables that customize
+the components that are deployed or alter some script behavior (such as to
 enable debug output). All values in `user.env` files are exported as environment
 variables available to the scripts. A `#` as the first character in a line
```

Comment:
The sentence related to comments should be sometnhing like: _"Any line with a '#' as the first character will be treated as a comment and ignored."_

Reply:
Changed to "Any line whose first character is `#` is treated as a comment and ignored."